tests/test_util.py

```diff
@@ -160,6 +160,54 @@ def test_bipartite_match():
         assert v in G[k] or k in G[v]
 
 
+def test_validate_intervals():
+    # Test for ValueError when interval shape is invalid
```

Comment:
Because I saw it was missing ;). I suppose it should have been part of a separate pr.